### PR TITLE
Deletes crappy fallout food.dm, moves everything to meat.dm.

### PR DESCRIPTION
**Pull request by skateboardjones**
## Description

Right, so everything that was in code/modules/fallout/obj/food_and_drinks/food.dm has now been moved over to food_and_drinks/food/snacks/meat.dm. Duplicates were removed, like deathclaw meat and gecko steaks, since they went unused and are inferior to the ones we already have and the ones that already spawn in-game.
## Motivation and Context

It's absolutely, horribly confusing to constantly switch between files, only to find duplicate items that aren't really duplicates, but have the same exact path. Plus they don't even spawn in. Jando asked me to do this, I swear.
## How Has This Been Tested?

Tested it on a local server. Compiled with no errors, nothin'. Crafted all the moved wasteland food, found and spawned in the junkfood items like BlamCo, etc. All worked as intended. Butchered a feral dog, meat is cool and good. Deathclaw steaks working, everything works. 
## Screenshots (if appropriate):
N/A
## Changelog (necessary)
:cl:
food.dm deleted
food.dm contents moved to meat.dm
/:cl:

### CODE REVIEW

**Comment by Laus4Deus** [comment]
It was in fact, me who tasked them with it.

File: code/modules/food_and_drinks/food/snacks/meat.dm [hunk 1]
```diff
@@ -530,3 +564,323 @@
...
+	bonus_reagents = list("radium" = 1, "vitamin" = 3)
+	list_reagents = list("nutriment" = 35)
+	filling_color = "#B22222"
+	trash = /obj/item/trash/f13/dog
+
+/obj/item/reagent_containers/food/snacks/f13/mre
+	name = "MRE"
+	desc = "The Meal, Ready-to-Eat : commonly known as the MRE - is a self-contained, individual field ration in lightweight packaging.<br>It's commonly used by military groups for service members to use in combat or other field conditions where organized food facilities are not available."
+	icon_state = "mre"
+	bonus_reagents = list("silver_sulfadiazine" = 10, "tricordrazine" = 10, "vitamin" = 2)
```
Comment (Trilbyspaceclone) [inline code/modules/food_and_drinks/food/snacks/meat.dm:741]: iirc sliver sulfadizine dosnt heal when ate, its based on touch/vaper*

File: code/modules/food_and_drinks/food/snacks/meat.dm [hunk 2]
```diff
@@ -530,3 +564,323 @@
...
+    icon_state = "meatbun"
+    bonus_reagents = list("vitamin" = 5, "tricordrazine = 5")
+    list_reagents = list("nutriment" = 30)
+    filling_color = "B#22222"
+
+/obj/item/reagent_containers/food/snacks/f13/deathclawomelette
+    name = "Deathclaw Omelette"
+    desc = "A delicious omelette made from one big deathclaw egg. Hope you're not allergic."
+    icon_state = "deathclawomlette"
+    bonus_reagents = list ("vitamin" = 5, "omnizine" = 40, "dexalin" = 10)
```
Comment (Trilbyspaceclone) [inline code/modules/food_and_drinks/food/snacks/meat.dm:795]: eating this is lethal do to omnizine od

**Comment by scarwolff** [comment]
do dah snack vendor still work tho

File: code/modules/food_and_drinks/food/snacks/meat.dm (hunk 1 above)
Comment (Laus4Deus) [inline code/modules/food_and_drinks/food/snacks/meat.dm:741]: `if(method in list(INGEST, VAPOR, INJECT))
			M.adjustToxLoss(0.5*reac_volume)`
Why **TF** the russians put a literal poison inside the MRE?????
Now, I am not 100% sure how `bonus_reagents` work compared to `list_reagents` but please, go safe with all the chems.

I will read up on the matter, plase do so as well

Comment (Laus4Deus) [reply code/modules/food_and_drinks/food/snacks/meat.dm:741]: Okay so `bonus_reagents` are something that appear when an item is cooked (made by a player) opposed to being spawned in by staff or being loot put there by a mapper.

`list_reagents` is the thing that is always there in the food.
So, please, so remove those that are wrong or toxic and tweak those that may overdose. And ask any doubts! Such as deathclaw meat, able to overdose!!

**Comment by Laus4Deus** [comment]
I am thinking real hard about deathclaw raw meat.

`/obj/item/reagent_containers/food/snacks/meat/slab/deathclaw
	desc = "A chemical soaked slab of deathclaw meat."
	icon_state = "goliathmeat"
	list_reagents = list("nutriment" = 3, "cooking_oil" = 2, "bicaridine" = 10, "kelotane" = 10, "antitoxin" = 10, "tricordrazine" = 10)`

`/obj/item/reagent_containers/food/snacks/meat/slab/deathclaw
	name = "deathclaw meat"
	desc = "A slab of hard but delicious deathclaw meat."
	list_reagents = list("nutriment" = 9, "vitamin" = 9)
	bitesize = 6 //Big slabs of meat from a massive creature
	filling_color = "#FA8072"
	cooked_type = /obj/item/reagent_containers/food/snacks/meat/steak/deathclaw
	slice_path = null`

This are the two right? The first one is in the fallout .dm you are deleting and the second in the right path. But in game, you have a meat that's bitesize 6 (the second meat) and has the healing reagents to (the first meat). What it seems to happen, with the magic of byond is that the two just **FUSE**.

*sigh* This needs more testing, in game, than we expected.

**Comment by skateboardjones** [comment]
snack vendors work, yes @scarwolff

**Comment by judgex** [comment]
I am reverting this at Jando's request.  Resubmit when issues are resolved, please!

**Pull request merged**